convert2rhel/unit_tests/actions/post_conversion/remove_tmp_dir_test.py

```diff
@@ -0,0 +1,81 @@
+# Copyright(C) 2024 Red Hat, Inc.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+__metaclass__ = type
+
+import logging
+import os
+
+import pytest
+
+from convert2rhel import unit_tests, utils
+from convert2rhel.actions.post_conversion import remove_tmp_dir
+
+
+@pytest.fixture
+def remove_tmp_dir_instance():
+    return remove_tmp_dir.RemoveTmpDir()
+
+
+def test_remove_tmp_dir(remove_tmp_dir_instance, monkeypatch, tmpdir, caplog):
+    caplog.set_level(logging.INFO)
+    path = str(tmpdir)
+    monkeypatch.setattr(remove_tmp_dir_instance, "tmp_dir", path)
+    assert os.path.isdir(path)
```

Comment:
Not sure if we need this assert here. `tmpdir` is supposed to always return a directory.